Sources/FluidAudio/TextToSpeech/Kokoro/Kokoro.swift

```diff
@@ -0,0 +1,284 @@
+import CoreML
```

Comment:
As discussed, a coupel problems right now 

- short text isn't handled well, likely due to padding and not masking properly in the models
- we aren't truncating the short text, and outputting 15s all the time
- we need to also batch text thats long and stitch them together